globals.go

```diff
@@ -490,7 +490,11 @@ func IsGitHubPatchRequester(requester string) bool {
 }
 
 // Permissions-related constants
-var AclCheckingIsEnabled = false
+var PermissionSystemDisabled = false // don't ever access this directly except from testutil
```

Comment:
An alternative would be to make this private, and have both the getter and setter for it in globals.go, so that way you have to explicitly call a function with a scary name to disable auth. I suppose the advantage of your approach is that someone would have to explicitly use testutil code to use the setter. On the other hand they can access this variable directly, so putting the setter somewhere else seems less useful.

Reply:
I'd put settings like this in Environment rather than in global state

Reply:
Moved to the env. The reason I have the setter in testutil is because that alerts if you call it from production code, which I want